### ISSUE TITLE
[TF2] [Feature Request] requesting tf_[language]_mod.txt as a way of overriding text without needing to update it every time localization files are updated

### ISSUE THREAD

**Issue by Whurrhurr** [ValveSoftware/Source-1-Games#3725]
title might be a bit confusing, but what the feature request basically is, is just "give users a way to modify a file like tf_english in their custom folder/vpk without it being broken by an update adding new items to the game" what this would let you do is rename something like the horsemann's name in his spawn announcement text from "The Horseless Headless Horsemann has appeared!" to "The Horsemann has appeared!" without needing to copy the entirety of tf_english.txt to change 1 line in the text file. and when an update to the localization files happens it because for example, if the game were to get updated and a new cosmetic gets added and tf_english.txt has the string for it as #TFCosmeticMedalPromo with the text that's normally seen in-game as "Team Fortress 2 Promo Medal" the user with their modded tf_english.txt will see #TFCosmeticMedalPromo instead of "Team Fortress 2 Promo Medal" because the modded tf_english.txt in the custom folder/vpk hasn't been updated to match the recent localization files update.

TL:DR
give users a way to override strings by using tf_{language]_mod.txt that lets them override whatever lines of text they want instead of modifying the entire tf_[language].txt file they want to modify that will become out of date/broken anytime a localization files update updates that file.
